### ISSUE TITLE
turf.bboxPolygon returns coordinates as strings, should return floats

### ISSUE THREAD

**Issue by knutole** [Turfjs/turf#1119]
The following code,
```
var temp2 = ["9.88311559495011", "60.7769672321184", "9.99246193731272", "60.7954097346711"]
var bboxPolygon = turf.bboxPolygon(temp2);
```
returns GeoJSON with coordinates as `string`, instead of `float`. In other words, invalid GeoJSON:
```
"{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[["9.88311559495011","60.7769672321184"],["9.99246193731272","60.7769672321184"],["9.99246193731272","60.7954097346711"],["9.88311559495011","60.7954097346711"],["9.88311559495011","60.7769672321184"]]]},"properties":{}}"
```

Should return coordinates as float, even though input is string.

**Comment by DenisCarriere** [comment]
@knutole Thanks for sharing, however at the moment TurfJS will not be supporting string data inputs since the documentation & TypeScript definitions strictly enforce the correct input types, it would be up to your application to submit the correct BBox type which would be an Array of Numbers.

For an easy fix you can convert an Array of Strings to Number by doing the following:

```js
const temp2 = ["9.8", "60.7", "9.9", "60.7"]
const bbox = temp2.map(str => Number(str))
const bboxPolygon = turf.bboxPolygon(bbox);
```

**Comment by DenisCarriere** [comment]
> Should return coordinates as float, even though input is string.

Ops... misread the issue, you mean this is a valid input? This should fail the input validation... 🤔 ..

**Comment by DenisCarriere** [comment]
Oh... I see.. `@turf/bbox-polygon` has no input validation... 🤦‍♂️ 

https://github.com/Turfjs/turf/blob/master/packages/turf-bbox-polygon/index.js#L17-L30

**Comment by DenisCarriere** [comment]
Will be fixed in the next minor release `v5.1`
